### PR TITLE
Add Lab test_type_code to all 1-20 labs in the kafka and message parser schemas

### LINKED ISSUE #565: Update Message Parser and Kafka schema's to include Lab Codes

### PR DESCRIPTION
**Pull request by BradySkylight**
# PULL REQUEST

## Summary
Ensure that 1-20 columns exist for all the lab results from FHIR in the message parser and kafka schemas.  Also added a new test_type_code, which will contain either the LOINC or SNOMED codes (depending on the STLT, for LAC it will be LOINC) that we can use for filtering out all labs but COVID tests.

## Related Issue
Fixes #565 

## Additional Information
Dan will be supplying the filter codes based upon his work with the spreadsheet sent by LAC.

### CODE REVIEW

**Review by m-goggins — CHANGES_REQUESTED** [review]
Looks good to me!

**Review by m-goggins — APPROVED** [review]
Accidentally hit "request changes" instead of "approve"

File: containers/message-parser/app/default_schemas/ecr.json [hunk 1]
```diff
@@ -28,73 +28,163 @@
...
     "specimen_type_1": "Bundle.entry.resource.where(resourceType='Observation').where(category.coding.code='laboratory')[0].extension.where(url='http://hl7.org/fhir/R4/specimen.html').extension.where(url='specimen source').valueString",
     "performing_lab_1": "Bundle.entry.resource.where(resourceType='Observation').where(category.coding.code='laboratory')[0].performer.display",
     "specimen_collection_date_1": "Bundle.entry.resource.where(resourceType='Observation').where(category.coding.code='laboratory')[0].extension.where(url='http://hl7.org/fhir/R4/specimen.html').extension.where(url='specimen collection time').valueDateTime",
     "result_date_1": "Bundle.entry.resource.where(resourceType='Observation').where(category.coding.code='laboratory')[0].effectiveDateTime",
     "test_type_2": "Bundle.entry.resource.where(resourceType='Observation').where(category.coding.code='laboratory')[1].code.coding.display",
+    "test_type_code_2": "Bundle.entry.resource.where(resourceType='Observation').where(category.coding.code='laboratory')[1].code.coding.code",
```
Comment (emmastephenson) [inline containers/message-parser/app/default_schemas/ecr.json:39]: This is just a FHIR curiosity question, but what happens if the resource path doesn't exist? i.e. if there's only 2 tests total in the bundle?

**Review by emmastephenson — APPROVED** [review]
Looks great!

File: containers/message-parser/app/default_schemas/ecr.json (hunk 1 above)
Comment (BradySkylight) [reply containers/message-parser/app/default_schemas/ecr.json:39]: It will return empty or ""

**Pull request merged**